From c20834d8c21e3c5a00840d9865dfec8a6acce15a Mon Sep 17 00:00:00 2001
From: Alaa Hleihel <ahleihel@redhat.com>
Date: Tue, 19 May 2020 07:48:37 -0400
Subject: [PATCH 216/312] [netdrv] net/mlx5: make the symbol 'ESW_POOLS' static

Message-id: <20200519074934.6303-7-ahleihel@redhat.com>
Patchwork-id: 310506
Patchwork-instance: patchwork
O-Subject: [RHEL8.3 BZ 1663246 06/63] net/mlx5: make the symbol 'ESW_POOLS' static
Bugzilla: 1663246
RH-Acked-by: Marcelo Leitner <mleitner@redhat.com>
RH-Acked-by: Jarod Wilson <jarod@redhat.com>
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Kamal Heib <kheib@redhat.com>

Bugzilla: http://bugzilla.redhat.com/1663246
Upstream: v5.6-rc1
Conflicts:
 - drivers/net/ethernet/mellanox/mlx5/core/eswitch_offloads_chains.c
   Context diff due to resolution of previous merge conflict at the
   declaration of ESW_POOLS array.

commit e15cf98ee8a76472144a19a24ca73d26fefa5237
Author: Chen Wandun <chenwandun@huawei.com>
Date:   Mon Jan 20 20:41:53 2020 +0800

    net/mlx5: make the symbol 'ESW_POOLS' static

    Fix the following sparse warning:
    drivers/net/ethernet/mellanox/mlx5/core/eswitch_offloads_chains.c:35:20: warning: symbol 'ESW_POOLS' was not declared. Should it be static?

    Fixes: 39ac237ce009 ("net/mlx5: E-Switch, Refactor chains and priorities")
    Signed-off-by: Chen Wandun <chenwandun@huawei.com>
    Acked-by: Paul Blakey <paulb@mellanox.com>
    Signed-off-by: Saeed Mahameed <saeedm@mellanox.com>

Signed-off-by: Alaa Hleihel <ahleihel@redhat.com>
Signed-off-by: Frantisek Hrbata <fhrbata@redhat.com>
---
 drivers/net/ethernet/mellanox/mlx5/core/eswitch_offloads_chains.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/net/ethernet/mellanox/mlx5/core/eswitch_offloads_chains.c b/drivers/net/ethernet/mellanox/mlx5/core/eswitch_offloads_chains.c
index d569969afd9d..cdf435cd08fb 100644
--- a/drivers/net/ethernet/mellanox/mlx5/core/eswitch_offloads_chains.c
+++ b/drivers/net/ethernet/mellanox/mlx5/core/eswitch_offloads_chains.c
@@ -32,10 +32,10 @@
  * pools.
  */
 #define ESW_SIZE (16 * 1024 * 1024)
-const unsigned int ESW_POOLS[] = { 4 * 1024 * 1024,
-				   1 * 1024 * 1024,
-				   64 * 1024,
-				   128 };
+static const unsigned int ESW_POOLS[] = { 4 * 1024 * 1024,
+					  1 * 1024 * 1024,
+					  64 * 1024,
+					  128 };
 
 struct mlx5_esw_chains_priv {
 	struct rhashtable chains_ht;
-- 
2.13.6

